planb/cli.py

```diff
@@ -80,8 +84,48 @@ def create(regions: list,
     create_cluster(options=locals())
 
 
-sns_topic_help = 'SNS topic name to send Auto-Recovery notifications to'
-sns_email_help = 'Email address to subscribe to Auto-Recovery SNS topic'
+@cli.command()
+@click.option('--from-region', type=str, required=True)
+@click.option('--to-region', type=str, required=True)
+@click.option('--cluster-name', type=str, required=True)
+@click.option('--ring-size', type=int, required=True)
+@click.option('--dc-suffix', default='', type=str)
```

Comment:
I propose to use click callbacks to validate the input to check for allowed chars.

Reply:
Do you have a list of allowed chars for this specific option?

Reply:
We use it for DNS as well: http://docs.aws.amazon.com/Route53/latest/DeveloperGuide/DomainNameFormat.html

I'm not shure what cassandra will support. Reducing to the valid chars for names (alpha-numeric characters and underscores) in C* gives reasonable subset of both.

We should avoid uppercases.